internal/provisioner/helm/controllers/bundledeployment_controller.go

```diff
@@ -286,7 +323,7 @@ const (
 	stateError        releaseState = "Error"
 )
 
-func (r *BundleDeploymentReconciler) getReleaseState(cl helmclient.ActionInterface, obj metav1.Object, chrt *chart.Chart, values chartutil.Values) (*release.Release, releaseState, error) {
+func (r *BundleDeploymentReconciler) getReleaseState(cl helmclient.ActionInterface, obj metav1.Object, chrt *chart.Chart, values chartutil.Values, post *postrenderer) (*release.Release, releaseState, error) {
```

Comment:
We may want/need a tech debt ticket here given the number of parameters for this method.

Reply:
OK.  I'll add a TODO comment.